src/eei.cpp

```diff
@@ -529,6 +577,13 @@ namespace HeraVM {
       }
 
       evm_result call_result;
+
+      if (import->base == Name("call") && !context->fn_table->account_exists(context, &call_message.destination))
+        takeGas(GasSchedule::callnewacct);
+      if (!isZeroUint256(call_message.value))
+        takeGas(GasSchedule::valuetransfer);
+      takeGas(gas);
```

Comment:
I think this actually should use `call_message.gas` because the all-but-1/64th rule applies to this too.